pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -136,6 +136,9 @@ type ClusterStatus struct {
 	// GitVersion of the kubernetes cluster, this field is populated by cluster controller
 	KubernetesVersion string `json:"kubernetesVersion,omitempty"`
 
+	// GitVersion of the /kapis/version api response, this field is populated by cluster controller
+	KubesphereVersion string `json:"kubesphereVersion,omitempty"`
```

Comment:
Actually, I suggest make it to `KubeSphereVersion string `json:"kubeSphereVersion,omitempty"``

`kube` and `sphere` are two words.

Reply:
Sorry for the misunderstanding. Done.

Reply:
No worries.